### ISSUE TITLE
Docker locks up in my_init

### ISSUE THREAD

**Issue by loleg** [scragg0x/realms-wiki#36]
When running the current Docker build, the VM endlessly cycles in `/sbin/my_init`, with log messages like:
```
./run: line 33: setuid: command not found
./run: line 34: setgid: command not found
./run: line 39: respawn: command not found
```
It seemed something is amiss with the base image? I tried swapping out `realms-base` for:
```
FROM phusion/baseimage:0.9.15
RUN useradd -M deploy
```
But that had no effect.

**Comment by CristianCantoro** [comment]
Just a quick note. After spending some time trying to figure out this problem I have found this bug. It seems that at [`realms-wiki`@DockerHub](https://registry.hub.docker.com/u/realms/realms-wiki/tags/manage/) the latest version availabe is `0.5.1` so I think this has not been fixed yet.

**Comment by theherk** [comment]
Thank you @CristianCantoro, that did the trick. Just a note you'll need not only the Dockerfile, but also the shell script. You can get these with:

    wget https://raw.githubusercontent.com/scragg0x/realms-wiki/master/docker/Dockerfile
    wget https://raw.githubusercontent.com/scragg0x/realms-wiki/master/docker/realms-wiki.sh

**Comment by CristianCantoro** [comment]
@theherk, of course you can clone the repo and modify directly the Dockerfile.

**Comment by eric-schleicher** [comment]
Sorry, I'm a little slow on the uptake  what exactly needs to be changes in the Dockerfile and/or realms-wiki/sh file to get passed the problem that @loleg  brought to attention?

i 'wgetted' the files that @theherk  put references to as well as cloned the repo locally both with the same result.

```sh

user@structr-sandbox:~$ docker run -t -i -p 8000:5000 user/realms-wiki
*** Running /etc/my_init.d/00_regen_ssh_host_keys.sh...
No SSH host key available. Generating one...
Creating SSH2 RSA key; this may take some time ...
Creating SSH2 DSA key; this may take some time ...
Creating SSH2 ECDSA key; this may take some time ...
Creating SSH2 ED25519 key; this may take some time ...
invoke-rc.d: policy-rc.d denied execution of restart.
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 96
./run: line 3: limit: command not found
./run: line 5: respawn: command not found
./run: line 7: description: command not found
./run: line 8: author: command not found
./run: line 10: chdir: command not found
./run: line 33: setuid: command not found
./run: line 34: setgid: command not found
./run: line 39: respawn: command not found
```

Superficially this looks like the user running the script in the container lacks suffecient priveledges (to fee the commands?  (limit, respawn, etc)

Please advise, this looks like a great wiki, excited to give it q real whirl.

**Comment by scragg0x** [comment]
issue should be fixed in b30c140e86e52da1aee4b1e6432428d933161480.  I am attempting to deploy now.

**Comment by theherk** [comment]
@eric-schleicher, The question is from which source was the image that you are running was built. You are running `user/realms-wiki`, but was that built from the repository online or from the files that your pulled down?
